cmd/template/cluster/flag.go

```diff
@@ -85,6 +87,7 @@ func (f *flag) Init(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&f.Region, flagRegion, "", "Installation region (e.g. eu-central-1 or westeurope).")
 	cmd.Flags().StringVar(&f.Release, flagRelease, "", "Tenant cluster release.")
 	cmd.Flags().StringSliceVar(&f.Label, flagLabel, nil, "Tenant cluster label.")
+	cmd.Flags().StringVar(&f.ReleaseBranch, flagReleaseBranch, "", "Release branch to use.")
```

Comment:
should the default be `master`?

Reply:
We have defaulting here https://github.com/giantswarm/kubectl-gs/blob/e2c4967512482be13b529f64f3ac0d4520cb1d47/pkg/release/release.go#L43

Do you think we need to specify the default here, as well?

Reply:
I don't know. I was thinking from the customer point of view. Without the default here, the customer won't know the branch, or will think it needs to pass a branch (since the default is empty). WDYT?

Reply:
I also specified the branch in the flag defaults. We shouldn't rely on the external `pkg/release` for defaulting.

Reply:
Sounds good! Thanks.